packages/zwave-js/src/lib/commandclass/EntryControlCC.ts

```diff
@@ -204,65 +268,56 @@ export class EntryControlCCAPI extends CCAPI {
 			this.commandOptions,
 		);
 		if (response) {
-			return pick(response, ["keyCachedSize", "keyCachedTimeout"]);
+			return pick(response, ["keyCacheSize", "keyCacheTimeout"]);
 		}
 	}
 
 	protected [SET_VALUE]: SetValueImplementation = async (
 		{ property },
 		value,
 	): Promise<void> => {
-		if (property !== "keyCachedSize" && property !== "keyCachedTimeout") {
+		if (property !== "keyCacheSize" && property !== "keyCacheTimeout") {
 			throwUnsupportedProperty(this.ccId, property);
 		}
 		if (typeof value !== "number") {
 			throwWrongValueType(this.ccId, property, "number", typeof value);
 		}
 		const valueDB = this.endpoint.getNodeUnsafe()!.valueDB;
 
-		let keyCachedSize = value;
-		let keyCachedTimeout: number = value;
-		if (property === "keyCachedSize") {
-			const oldKeyCachedTimeout = valueDB.getValue<number>(
-				getKeyCachedTimeoutStateValueID(this.endpoint.index),
-			);
-			if (oldKeyCachedTimeout == undefined) {
-				throw new ZWaveError(
-					`The "keyCachedSize" property cannot be changed before the key cached timeout is known!`,
-					ZWaveErrorCodes.Argument_Invalid,
-				);
-			}
-			keyCachedTimeout = oldKeyCachedTimeout;
-		} else {
-			const oldKeyCachedSize = valueDB.getValue<number>(
-				getKeyCachedSizeStateValueID(this.endpoint.index),
+		let keyCacheSize = value;
+		let keyCacheTimeout = 2;
+		if (property === "keyCacheTimeout") {
+			keyCacheTimeout = value;
+
+			const oldKeyCacheSize = valueDB.getValue<number>(
+				getKeyCacheSizeStateValueID(this.endpoint.index),
 			);
-			if (oldKeyCachedSize == undefined) {
+			if (oldKeyCacheSize == undefined) {
 				throw new ZWaveError(
-					`The "keyCachedTimeout" property cannot be changed before the key cached size is known!`,
+					`The "keyCacheTimeout" property cannot be changed before the key cache size is known!`,
 					ZWaveErrorCodes.Argument_Invalid,
 				);
 			}
-			keyCachedSize = oldKeyCachedSize;
+			keyCacheSize = oldKeyCacheSize;
 		}
-		await this.setConfiguration(keyCachedSize, keyCachedTimeout);
+		await this.setConfiguration(keyCacheSize, keyCacheTimeout);
 	};
 
 	protected [POLL_VALUE]: PollValueImplementation = async ({
 		property,
 	}): Promise<unknown> => {
 		switch (property) {
-			case "dataTypeSupported":
-			case "eventTypeSupported":
-			case "keyCachedSizeMin":
-			case "keyCachedSizeMax":
-			case "keyCachedTimeoutMin":
-			case "keyCachedTimeoutMax":
-				return (await this.getEventSupported())?.[property];
-			case "keySupported":
-				return (await this.getKeySupported())?.[property];
-			case "keyCachedSize":
-			case "keyCachedTimeout":
+			case "supportedDataTypes":
+			case "supportedEventTypes":
+			case "minKeyCacheSize":
+			case "maxKeyCacheSize":
+			case "minKeyCacheTimeout":
+			case "maxKeyCacheTimeout":
+				return (await this.getEventCapabilities())?.[property];
+			case "supportedKeys":
+				return this.getSupportedKeys();
```

Comment:
Sorry, it was late last night when I reviewed. Most of these shouldn't be public values, so they shouldn't be pollable with this API. `minKeyCacheSize` etc should be included in metadata, `supportedDataTypes, supportedEventTypes, supportedKeys` should be known after the interview and not change.